frontend/micro-ui/web/micro-ui-internals/packages/modules/obps/README.md

```
@@ -22,10 +22,10 @@ frontend/micro-ui/web/package.json
 
 ## Changelog
 
-### Summary for Version [1.7.10] - 2025-02-13
+### Summary for Version [1.7.35] - 2025-03-20
 
 Started using inboxv2 on employee side
 
 ### Contributors
 
-[jagankumar-egov] [hari-egov] [mukund-egov]
+[jagankumar-egov] [hari-egov] [mukund-egov]
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)